src/pipeline.ts

```diff
@@ -202,16 +203,18 @@ export async function searchForLocalTorrentByCriteria(
 	searchees.map((s) => (s.label = Label.WEBHOOK));
```

Comment:
I wanted to avoid putting it the the lower level functions as it would color everything. It would also mean that each searchee must be created for one of the label's purpose when it can be used outside of those. I'll change the modifying to replace.